sphinx_rtd_theme/static/js/theme.js

```diff
@@ -170,7 +187,7 @@ function ThemeNav () {
 module.exports.ThemeNav = ThemeNav();
 
 if (typeof(window) != 'undefined') {
-    window.SphinxRtdTheme = { StickyNav: module.exports.ThemeNav };
+    window.SphinxRtdTheme = { Navigation: module.exports.ThemeNav };
```

Comment:
This likely breaks on RTD. Has this been tested?

Reply:
This also comes from #519. The theme code that calls this has been [changed accordingly](https://github.com/rtfd/sphinx_rtd_theme/pull/519/files#diff-1336f80431f476a232e3c48abb277c51).
It was tested on a local build, but I don't know if it has been tested on RTD. I can't imagine this function being called from outside of the theme though. A [search in the RTD source](https://github.com/rtfd/readthedocs.org/search?utf8=%E2%9C%93&q=StickyNav&type=) does not turn up anything.

Reply:
At some point we were overriding or referencing `StickyNav` in RTD. So this would be worth testing.